samples/drivers/uart/stm32/single_wire/boards/stm32f3_disco.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2021 Jonathan Hahn
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	 aliases {
+		single-line-uart1 = &usart2;
+		single-line-uart2 = &uart4;
+	};
+};
+
+&usart2 {
+	single-wire;
```

Comment:
Not functionally  required, but educating.
```suggestion
	pinctrl-0 = <&usart2_tx_pa2>;
	single-wire;
```

Reply:
ah didn't look into it yet, but now I get what you meant.. I thought of something in c code which disallows usage of anything more than the tx pin

Reply:
done